ingestion/src/metadata/cli/docker.py

```diff
@@ -276,9 +275,13 @@ def run_docker(  # pylint: disable=too-many-branches
                 "Stopping docker compose for OpenMetadata and removing images, networks, volumes..."
             )
             logger.info("Do you want to Delete the docker mounted volumes from host")
```

Comment:
```suggestion
            logger.info("Do you want to delete the MySQL docker volume with the OpenMetadata data?")
```
Should we ask for deleting the MySQL data since it is the only local volume mounted? WDYT?

Reply:
makes sense, let me make that change